udata/tests/api/test_dataservices_api.py

```diff
@@ -3,21 +3,41 @@
 import pytest
 from flask import url_for
 
+import udata.core.organization.constants as org_constants
 from udata.core.dataservices.factories import DataserviceFactory
 from udata.core.dataservices.models import Dataservice
 from udata.core.dataset.factories import DatasetFactory, LicenseFactory
 from udata.core.organization.factories import OrganizationFactory
 from udata.core.organization.models import Member
 from udata.core.user.factories import UserFactory
 from udata.i18n import gettext as _
-from udata.tests.helpers import assert200, assert_redirects
+from udata.tests.helpers import assert200, assert400, assert_redirects
```

Comment:
Very slightly bothered by the non-pythonic naming of those functions with no underscore, but that's just me 😆

Reply:
yeah, it's a mix between pythonic and "unittest-ic" 😬